src/quests.c

```diff
@@ -2059,14 +2064,35 @@ void GenerateQuestFlavorText(s32 questId)
 }
 void UpdateQuestFlavorText(s32 questId)
 {
-	StringCopy(gStringVar1, sSideQuests[questId].desc);
+	StringExpandPlaceholders(gStringVar1, GetQuestDesc(questId));
 }
 void PrintQuestFlavorText(s32 questId)
 {
 	QuestMenu_AddTextPrinterParameterized(1, 2, gStringVar3, 40, 19, 5, 0, 0,
 	                                      4);
 }
 
+static const u8 *GetQuestLocation(s32 questId)
+{
+    switch (questId) {
+        case QUEST_3:
+            return gTable_Quest3Maps[VarGet(VAR_UNUSED_0x404E)]; // uses first unused VAR in include/constants/vars.h
```

Comment:
Does this mean the user needs to load that variable with the desired state before using these functions? Or is this paired to the variable for the quest itself?